third_party/proto/gogoproto/gogo.proto

```diff
@@ -0,0 +1,147 @@
+// Protocol Buffers for Go with Gadgets
+//
+// Copied from https://github.com/gogo/protobuf/blob/master/gogoproto/gogo.proto
+//
+// Copyright (c) 2013, The GoGo Authors. All rights reserved.
```

Comment:
Why do we have to copy this file?

Reply:
Since we are not using GOPATH anymore then it was required to use `vendor` dir, but if we were to go with vendor then we would need devs that are fetching these files to also fetch the vendor file, at least the gogoproto dep & the buf file would be something along the lines of
```
build:
  roots:
    - .
lint:
  ignore:
    - conformance
    - gogoproto
    - jsonpb
    - proto
    - protobuf
    - protoc-gen-gogo
    - test
    - vanity
breaking:
  ignore:
    - conformance
    - gogoproto
    - jsonpb
    - proto
    - protobuf
    - protoc-gen-gogo
    - test
    - vanity
``` 
or
```
build:
  roots:
    - .
  excludes:
    - vendor/github.com/gogo/protobuf/conformance
    - vendor/github.com/gogo/protobuf/jsonpb
    - vendor/github.com/gogo/protobuf/proto
    - vendor/github.com/gogo/protobuf/protobuf
    - vendor/github.com/gogo/protobuf/protoc-gen-gogo
    - vendor/github.com/gogo/protobuf/test
    - vendor/github.com/gogo/protobuf/vanity
lint:
  ignore:
    - gogoproto
breaking:
  ignore:
    - gogoproto
```
instead of what it is currently. This is because buf lists out the all the proto files located in the repo and with vendor this includes all the gogo proto ones.

Reply:
Okay, thanks!